games-arcade/rocksndiamonds/additional-files/rocksndiamonds.rdef

```diff
@@ -0,0 +1,39 @@
+// Rocks 'n' Diamonds icon by Stephanie Fu
+
+resource app_signature "application/x-vnd.rocksndiamonds";
+
+resource app_flags B_EXCLUSIVE_LAUNCH;
+
+resource app_version {
+	major = 0,
+	middle = 0,
+	minor = 0,
+	
+	variety = B_APPV_DEVELOPMENT,
+	internal = 0;
+
+	short_info = "",
+	long_info = ""
```

Comment:
You might wish to replace
```
	short_info = "",
	long_info = ""
```
by
```
	short_info = "Rocks'n'Diamonds",
	long_info = "An arcade style game by Artsoft Entertainment"
```
I'm not sure if the string I'm suggesting for long_info is consistent with what is expected.
Maybe it should be "Copyright 2013, Artsoft Entertainment". Any ideas?